### ISSUE TITLE
Add bug IDs to the build stdout

### ISSUE THREAD

**Issue by zolyfarkas** [gleclaire/findbugs-maven-plugin#26]
Currently Output looks like:

```
BugInstance size is 1
Error size is 0
Total bugs: 1
Method com.pimco.popcorn.camel.rest.MyClass.method(String) declares RuntimeException in throws clause [package.MyClass] At MyClass.java:[line 16]
```

if would be nice if every bug instance the BUG_ID would be displayed as well. Currently when I have to figure that out, I have to go through the extra step of running the findbugs:gui.

this should be something easy to implement...

**Comment by gleclaire** [comment]
I have added the bug type to the end of the output

**Comment by zolyfarkas** [comment]
Brilliant! Thanks!

**Comment by sishbi** [comment]
Any idea when v3.0.3 will be released?  I would like this fix - I am currently using 3.0.2
Thanks, Simon.
